tests/e2e.sh

```diff
@@ -145,7 +145,13 @@ assert_dir_exists knative-serving
 # kpt fn Tests
```

Comment:
Probably want to merge from upstream to pull in fixes to existing e2e tests.

Reply:
Done